### ISSUE TITLE
Background embedding

### ISSUE THREAD

**Issue by wusize** [dyabel/detpro#30]
Hi!
You mentioned SoftBG in your paper. But I cannot find its implementation in the codes. What I [find](https://github.com/dyabel/detpro/blob/5b07329aa05d4ef54d0348e96ace44a85e7adae6/mmdet/models/roi_heads/standard_roi_head.py#L603) is that you used a learned background embedding and masked the logits of the novel classes.

**Comment by Fangyi-Chen** [comment]
Hi, Have you resolved the issue?  I am having the same question.
